src/rpc/handlers/AccountObjects.cpp

```diff
@@ -84,21 +78,23 @@ doAccountNFTs(Context const& context)
     ripple::uint256 const maskedMarker = marker & ripple::nft::pageMask;
 
     // Continue iteration from the current page:
-    while (true)
+    while (cp)
     {
         auto arr = cp->getFieldArray(ripple::sfNFTokens);
 
-        for (auto const& o : arr)
+        for (size_t i = 0; i < arr.size(); ++i)
         {
+            size_t idx = arr.size() - i - 1;
+            auto const& o = arr[idx];
```

Comment:
```suggestion
        for (auto itr = std::make_reverse_iterator(arr.end()); itr != std::make_reverse_iterator(arr.begin()); ++itr))
        {
            auto const& o = *itr;
```

Reply:
Why not just use `arr.rbegin()` and `arr.rend()`?

Reply:
Those methods aren't defined for `STArray`